### PR TITLE
Some metadata cleanup, including adding metadata.json

### PR DESCRIPTION
**Pull request by gregswift**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="http://app.review.ninja/rackerlabs/puppet-repose/pull/48" target="_blank">ReviewNinja

File: metadata.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
+    "name": "rackerlabs-repose",
+    "version": "0.6.4",
```
Comment (mwhahaha) [inline metadata.json]: 1.6.4

File: metadata.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    "author": "Rackspace",
+    "license": "Apache-2.0",
+    "summary": "Repose is an api middleware that provides authentication, filtering, ratelimitting and several other features, this deploys it.",
+    "source": "git@github.com/rackerlabs/puppet-repose.git",
```
Comment (mwhahaha) [inline metadata.json]: should probably be https://github.com/rackerlabs/puppet-repose.git

**Comment by gregswift** [comment]
@mwhahaha updated

**Comment by mwhahaha** [comment]
looks ok to me

**Pull request closed without merging**